www/about.html

```diff
@@ -24,7 +24,10 @@
     </header>
 
     <main>
-      <a href="/">← Return to the tracker</a>
+      <a href="/">
+        <span aria-hidden="true">←</span>
```

Comment:
I think CSS content is visible by a screenreader but I'd have to try it out to be 100% sure.